src/main/java/com/artipie/asto/AsyncContent.java

```diff
@@ -43,4 +43,31 @@
 
     @Override
     CompletionStage<? extends Content> get();
+
+    /**
+     * Failed async content.
+     * @since 0.3
```

Comment:
@olenagerasimova `@since` version is off, it should be `0.30`. Same in other classes added in this PR